app/grandchallenge/components/serializers.py

```diff
@@ -25,6 +27,36 @@
 logger = logging.getLogger(__name__)
 
 
+def convert_deserialized_civ_data(*, deserialized_civ_data):
+    """Takes deserialized CIV data and returns list of CIVData objects."""
```

Comment:
Isn't the data serialized at this point? E.g. it had been serialized to a dictionary and you want to create `CIVData` objects?